setup.py

```diff
@@ -321,7 +321,11 @@ def initialize_options(self) -> None:
           # Not sure how/if interface will change in db-facts, so
           # let's be conservative about what we're specifying for now.
           'db-facts>=4,<5',
-          'chardet',
+          # Version 4.0.0 is not yet compatible with requests, and pip
```

Comment:
This file, one of the 3 largest in the project, increased in size to 377 lines.  The total size of those files is now 3419 lines (target: 1147).  Is this file complex enough to refactor?